### ISSUE TITLE
BERT loss curves train/eval

### ISSUE THREAD

**Issue by jboelaert** [emilienschultz/pyactivetigger#968]
For many models (not all), we have a strange discrepancy between train and eval curves, see screenshot 

The problem here is that costs are not averaged but summed. So if there's 20% eval cases, then the training loss curve will be 4x higher than the eval one. 
@AXLMRIN says it's fixable with a loss function parameter

![Image](https://github.com/user-attachments/assets/f77b6b0f-2217-42a5-89c2-fefe490f58ed)
